client/src/components/AccountLayout.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { Box, Heading, Main } from 'grommet'
+import { useRouter } from 'next/router'
+import { SideNav } from './SideNav'
+import Header from './Header'
+
+export const AccountLayout = ({ children }) => {
+  const router = useRouter()
+  const links = [
+    {
+      text: 'Basic Information',
+      href: '/account/basic-information',
+      notifications: 0
+    },
+    {
+      text: 'Manage Resources',
+      href: '/account/manage-resources',
+      notifications: 0
+    },
+    {
+      text: 'Requests',
+      href: '/account/requests',
+      notifications: 0
+    },
+    {
+      text: 'Teams',
+      href: '/account/teams',
+      notifications: 0
+    },
+    {
+      text: 'Notifications',
+      href: '/account/notifications',
+      notifications: 4
```

Comment:
Is having 4 notifications hard coded in?

Reply:
yeah needs to be integrated still that will be done with account notifications section integration